services/download/src/main/java/app/coronawarn/server/services/download/DownloadServiceConfig.java

```diff
@@ -34,6 +34,9 @@
   @Min(0)
   @Max(14)
   private Integer efgsOffsetDays;
+  @Min(0)
+  @Max(28)
```

Comment:
who defined the 28?

Reply:
That's the same values as we have for the distributions service's retention policy